backend/src/main/java/com/crm/dto/request/CustomerRequest.java

```diff
@@ -0,0 +1,24 @@
+package com.crm.dto.request;
+
+import com.crm.exception.ErrorDict;
+import lombok.*;
+
+import javax.validation.constraints.NotEmpty;
+
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+public class CustomerRequest {
+
+    @NotEmpty(message = ErrorDict.CUSTOMER_NAME_INVALID)
+    private String name;
+    @NotEmpty(message = ErrorDict.CUSTOMER_SURNAME_INVALID)
+    private String surname;
+    @NotEmpty(message = ErrorDict.CUSTOMER_PHONE_INVALID)
+    private String phone;
```

Comment:
Please add additional validation for length without dash characters. Only consider Polish phone numbers without prefix